mod-source-record-manager-server/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.folio</groupId>
     <artifactId>mod-source-record-manager</artifactId>
-    <version>3.6.5-SNAPSHOT</version>
+    <version>3.7.0-SNAPSHOT</version>
```

Comment:
ah, i thought the release plugin incremented it in preparation for the next release. will revert this

Reply:
er no, it seems the other way around @RuslanLavrov 
https://wiki.folio.org/display/~marcjohnson/Marc%27s+Release+Process+for+Back+End+Modules

Reply:
do you mean that here we are increasing the minor version in compliance with 6th step on this wiki page?
It seems that this wiki page describes steps for a mainline release, but the changes in the PR are going to be released as Orchid bugfix (not as main Orchid release because it already has been done). In my understanding, it would be expected to increase the bugfix/patch version 3.6.**5** -> 3.6.**6** (instead of minor version), as it has been done in the PR for mod-data-import https://github.com/folio-org/mod-data-import/pull/299 in the news.md file (2.7.1 -> 2.7.2)

Reply:
Ah ok, I'll do this as a patch then. Thanks

Reply:
got it. Please don't forget to change the release version in the schema.json file as well (just a reminder). Thanks!